mail-filter/amavisd-new/amavisd-new-2.12.0-r1.ebuild

```diff
@@ -0,0 +1,198 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd
+
+DESCRIPTION="High-performance interface between the MTA and content checkers"
+HOMEPAGE="https://gitlab.com/amavis/amavis"
+SRC_URI="https://gitlab.com/amavis/amavis/-/archive/v${PV}/amavis-v${PV}.tar.bz2"
+
+LICENSE="GPL-2 BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="clamav courier dkim ldap mysql postgres qmail razor rspamd rspamd-https snmp spamassassin zmq"
+
+MY_RSPAMD_DEPEND="( dev-perl/JSON dev-perl/HTTP-Message dev-perl/LWP-UserAgent-Determined )"
+BDEPEND="acct-user/amavis"
```

Comment:
Hmm, are you sure about the BDPEND with this one?

Reply:
That actually appears to be a typo. I checked [the eclass docs](https://devmanual.gentoo.org/eclass-reference/acct-user.eclass/index.html) and it should be DEPEND, right?

Reply:
Yep, DEPEND makes more sense to me, because the user is needed in the target machine, not buildhost.